beginner/irods_beginner_training_2024.tex

```diff
@@ -1498,7 +1499,29 @@ \section{Using the Delay Queue}
 
 To demonstrate use of the delay queue, we will create a rule that utilizes \texttt{delay()}, submit it to the delay queue, and verify its output in the system log.
 
-First, create a file named \texttt{example\_delay.r} with the following content:
+First, we will activate the delay queue. Switch to the \texttt{rods} user. If you get stuck, refer to \autoref{sec:becomerods}.
+
+Next, activate the delay queue:
+
+\begin{lstlisting}
+$ iadmin set_delay_server $(hostname)
+\end{lstlisting}
```

Comment:
We should move this into the section following setup.

Reply:
End of 5.1, perhaps? Or did you have a spot in mind? It would save some space by not having to remind trainees to switch between `rods` and `alice`, so I don't mind this at all.

Reply:
i think it's okay where it is?   @korydraughn you are suggesting moving this setup/confirmation earlier in the document?

Reply:
Yes. If the doc has a section for setup, it should go there.

Reply:
it could go at the end of chapter 4... but we'd still have to 'switch to rods' and that doesn't happen until 'later'... so i think it's okay where it is.  it's already context specific and won't introduce a concept 'out of order'.

Reply:
the setup is all done as sudo - which is correct - so... i think i'm voting for leaving as it is now.

Reply:
The end of chapter 4 makes the most sense to me. Just set that delay server hostname following setup.

Reply:
that works for the slides, but feels... confusing in this document.

Reply:
Fair - okay - so Derek, please move the instructions and confirmation to the end of Chapter 4... and don't worry about switching users... just do it with sudo.   let's iterate on this quickly, so this thing can go to print.